### PR TITLE
Modulesync: use voxpupuli helper gems

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -1,38 +1,20 @@
...
-    hosts.each do |host|
-      if fact_on(host, 'osfamily') == 'RedHat'
-        # don't delete downloaded rpm for use with BEAKER_provision=no +
-        # BEAKER_destroy=no
-        on host, 'sed -i "s/keepcache=.*/keepcache=1/" /etc/yum.conf'
-        # refresh check if cache needs refresh on next yum command
-        on host, 'yum clean expire-cache'
-        # We need SCL as well for dynflow
-        if fact_on(host, 'operatingsystem') == 'CentOS'
-          host.install_package('centos-release-scl-rh')
```
Comment (ehelms) [inline spec/spec_helper_acceptance.rb:33]: We need this still

Comment (ekohl) [reply spec/spec_helper_acceptance.rb:33]: Yes, haven't gotten around to it. Working on a better pattern to combine modulesync and local module overrides.

**Comment by ekohl** [comment]
I first want to do a refactor of `foreman::repo` before I tackle this.

**Comment by ekohl** [comment]
Includes https://github.com/theforeman/puppet-foreman_proxy/pull/584 since that needs its own changelog entry.

**Comment by ekohl** [comment]
I believe this is now ready to be merged. Please only merge this fast forward or after #584 is merged for a proper history and changelog.

**Comment by ekohl** [comment]
Repo handling still needs some work. It errors out on the rails repo that's now empty. This is also a bug with 2.0 GA. https://github.com/theforeman/puppet-foreman/commit/475afc0a3809fba31eb358cbeae320afec23107b fixes this but acceptance tests only use released modules. I'll work on getting that in a stable release and rebase this.

**Comment by ekohl** [comment]
https://github.com/theforeman/puppet-foreman/pull/830

**Comment by ekohl** [comment]
Ok, I think this should now pass.

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**